.circleci/config.yml

```diff
@@ -108,6 +108,26 @@ jobs:
           command: |
             ./twistcli images scan --address $CONSOLE_URL --user=$USER --password=$PASSWORD --details << parameters.docker_image >>
 
+  release_bom:
+    docker:
+      - image: cimg/python:3.12
+    steps:
+      - checkout
+      - run:
+          name: Get Version from Git Tag
+          command: |
+            VERSION=$(git describe --tags --abbrev=0)
```

Comment:
This syntax doesn't work as expected. It is currently showing `v0.32.0` even though the newest tag is `v0.35.4`

Reply:
So it is working for the current branch it is triggered from, for example currently it is showing:
```
shubhamsingh@Shubhams-MacBook-Pro astro % git describe --tags --abbrev=0
v0.32.0
```

Reply:
sure, added the tag for now i am keeping the branches as well for testing, @danielhoherd

Reply:
changed it to use CIRCLE_TAG env variable